src/Views/AppInfoView.vala

```diff
@@ -472,6 +488,43 @@ public class AppCenter.Views.AppInfoView : Adw.NavigationPage {
             }
         }
 
+        var supports = package_component.get_supports ();
+        for (int i = 0; i < supports.length; i++) {
+            var relation = supports[i];
+
+            string title, description, icon_name;
+
+            switch (relation.get_value_control_kind ()) {
+                // An input method for users to control software
+                case GAMEPAD:
+                    switch (relation.get_kind ()) {
+                        case REQUIRES:
+                            title = _("Requires a Controller");
+                        case RECOMMENDS:
+                            title = _("Recommends a Controller");
+                        case SUPPORTS:
+                        default:
+                            title = _("Play With a Controller");
+                    }
+
+                    description = _("Such as a PlayStation, Xbox, or 8BitDo controller.");
+                    icon_name = "input-gaming-symbolic";
+                    break;
+
+                default:
+                    // unhandled
+                    debug (relation.get_value_control_kind ().to_string ());
```

Comment:
Can you give more context about this log? Maybe log a phrase like: "Unhandled kind: %s, %s".